cmd/ak/cmd/vars/set.go

```diff
@@ -32,16 +32,35 @@ var setCmd = common.StandardCommand(&cobra.Command{
 		}
 
 		var value string
-		if len(args) == 2 {
-			value = args[1]
+
+		isValueSet := len(args) > 1
+
+		if isValueSet {
+			if args[1] != "-" {
+				value = args[1]
+			} else {
+				const maxVarSize = 1 << 20 // 1MB
```

Comment:
1. Pull this outside the function?
2. Do we really want to allow 1 MiB vars? Maybe 1 KiB? I'd say even that would probably merit using a file in the project rather than a var, and FYI Windows env vars are limited to a few KiBs too